airbyte-integrations/connectors/source-apify-dataset/source_apify_dataset/spec.json

```diff
@@ -0,0 +1,20 @@
+{
+  "documentationUrl": "https://docsurl.com",
```

Comment:
Once this is merged, your docs page `apify-dataset.md` will be surfaced at  `https://docs.airbyte.io/integrations/sources/apify-dataset` so you can set this to that url